server/db/schemas/addresses.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE addresses (
+    user_id INT NOT NULL,
+    street_address VARCHAR(255) NOT NULL,
+    street_address_line_two VARCHAR(255),
+    city VARCHAR(255) NOT NULL,
+    province VARCHAR(255) NOT NULL,
+    postal_code VARCHAR(255) NOT NULL,
+    country VARCHAR(255) NOT NULL,
+    PRIMARY KEY (user_id, street_address),
```

Comment:
Replace compound primary key with `PRIMARY KEY (address_id)`. Currently there would be duplicated addresses.

Reply:
I also prefer to use primary keys instead of compound primary keys so ill add one, should make updating and deleting addresses easier.

Reply:
Pushed the update